src/bentoml/server.py

```diff
@@ -185,7 +185,7 @@ def __init__(
         self,
         bento: str | Bento | Tag,
         reload: bool = False,
-        production: bool = False,
+        production: bool = True,
```

Comment:
@aarnphm I'm updating HTTPServer and GrpcServer's production default to True, to match our cli default

Reply:
sg